mkdocs/docs/HPC/index.md

```diff
@@ -1,10 +1,32 @@
-# Welcome
+---
+hide: "toc"
+---
+# HPC Documentation
+
+---
+
+
+![image](img/ch0-hpc.png){: align=left style="height:383px;width:378px;border-radius:6px" }
+<!-- 1150 1134 -->
 
 Welcome to the {{ hpcinfra }} documentation!
```

Comment:
Maybe use this as title above (on line less next to picture), or just drop it since it doesn't really give useful information...